src/applications/facility-locator/components/search-results-items/Covid19Result.jsx

```diff
@@ -72,11 +73,19 @@ const Covid19Result = ({
               Schedule an appointment online
             </a>
           )}
+        {showCovidVaccineWalkInAvailabilityText && (
+          <strong className="vads-u-margin-bottom--2 vads-u-display--block">
```

Comment:
Noticing this is bold and a block display. Should it be a heading instead?

Reply:
Hmm, I think if we change to a heading tag, the font will change as well. Unless you know of a class that would make it more inline with the design.
Also, there was no mention of SEO related to that text. 
If there was, then I would make it a heading tag.